solidity-v1/dashboard/src/lib/keep/index.js

```diff
@@ -199,10 +207,26 @@ class Keep {
     this.keepToTStaking = new KeepToTStaking(
       this.thresholdStakingContract,
       this.thresholdKeepStakeContract,
+      this.simplePREApplicationContract,
       this.web3
     )
   }
 
+  _initializePREContract = async (web3) => {
```

Comment:
Let's call this function inside `initializeContracts`.

Reply:
a3ab04267c9d6acc69725ef3e469c8817810ac8e